components/blocks/badge/_badge.scss

```diff
@@ -0,0 +1,22 @@
+.badge {
+    font-size: 12px;
+    width: auto;
+    position: absolute;
+    padding: 5px 10px;
+    top: 0;
+
+    &--new {
+        background: $color-accent;
+        color: $color-white;
+    }
+
+    &--alert {
+        background: $color-aqua;
+        color: $color-white;
+    }
+
+    &--error {
+        background: $color-main1;
```

Comment:
This is true, I always make a mistake when writing this variable in the project and must look into a file to find what I need to write. This will end